cmake/cmake_uninstall.cmake.in

```diff
@@ -0,0 +1,22 @@
+# From: https://cmake.org/Wiki/CMake_FAQ
```

Comment:
it seems the code changed a bit (https://gitlab.kitware.com/cmake/community/-/wikis/FAQ#can-i-do-make-uninstall-with-cmake). Can we use the updated version?